sources/packages/api/src/route-controllers/supporting-user/supporting-user.aest.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { SupportingUserService } from "../../services";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { ApplicationSupportingUsersDTO } from "./models/supporting-user.dto";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("supporting-user")
```

Comment:
this could be changed to _aest/supporting-user_ to be aligned with the new routes and the _aest_ can be removed from the end of the GET method.

Reply:
mm, `aest` is part of the module right?

Reply:
Did you create the module already?

Reply:
Nevermind, I will assume that you will adjust in the next PR and remove the AEST from the end of the GET method, right?

Reply:
yup, that was my intension